staging_service/app.py

```diff
@@ -109,14 +110,21 @@ def _file_type_resolver(path: PathPy) -> FileTypeResolution:
 
 
 def _make_dts_file_resolver() -> Callable[[Path], FileTypeResolution]:
-    """Makes a DTS file resolver."""
+    """Makes a DTS file resolver.
+
+    This looks a little goofy, but it ensures that the DTS manifest schema file
+    only gets loaded once per API call, no matter how many DTS manifest files are
+    expected to be parsed. It also prevents having it stick around in memory.
```

Comment:
Yeah, it's a trade-off, right? I'm assuming it will very rarely get used. I don't think folks will be coming in every day to do a DTS transfer from outside, as opposed to uploading their own data.

But, like you said, it's tiny enough that it shouldn't have a large memory footprint.

But, on the other-other hand, it's still memory being gobbled that doesn't do anything useful.

Eh. How about this. At server startup, it loads the config, loads the schema, validates it to fail early, then just parks that in memory. Like you said, it's small, and I don't expect it'll get much more complex.

Reply:
That's exactly what I was thinking

Reply:
Done.

Reply:
:+1: